src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/Manager/MediaManagerTest.php

```diff
@@ -535,15 +529,15 @@ public function provideGetByIds()
         ];
     }
 
-    public static function provideSpecialCharacterFileName()
+    public static function provideSpecialCharacterFileName(): array
     {
         return [
             ['aäüßa', 'aäüßa', 'aaeuesa', ''],
             ['aäüßa.mp4', 'aäüßa', 'aaeuesa', '.mp4'],
         ];
     }
 
-    public static function provideSpecialCharacterUrl()
+    public static function provideSpecialCharacterUrl(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes